htdocs/compta/facture/card.php

```diff
@@ -2473,12 +2473,12 @@
 				setEventMessages($mesg, null, 'warnings');
 				$error++;
 				$result = -1;
-			} elseif ($progress < $percent) {
-				$mesg = '<div class="warning">'.$langs->trans("CantBeLessThanMinPercent").'</div>';
-				setEventMessages($mesg, null, 'warnings');
-				$error++;
-				$result = -1;
 			}
+		} elseif ($progress < $percent) {
```

Comment:
This is strange because you move a test about situation invoice out of a if that is related to situation invoice.
This means this test will be executed even for non situation invoice ?

Reply:
I do not understand your question !
The progress column is only displayed on the situation invoice!
Also, for now, the test is done only for situation credit note.

Reply:
When an invoice is not a situation invoice, i suppose that invoice with $object->situation_cycle_ref = 0 or invoice with $object->situation_cycle_ref is not defined.

Here, when  $object->situation_cycle_ref = 0, you make a test ($progress < $percent)   but $progress has a meaning only for situation invoice.